src/index.html

```diff
@@ -17,16 +17,33 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form"
+      action="#"
+      data-qa="big"
+      method="get"
+    >
+      <input
+        class="search search--big"
+        name="search-big"
+        data-qa="keypress"
+        type="text"
+        placeholder="Try “Los Angeles”"
+      />
```

Comment:
You have a `form` element here, but it doesn't contain any `label` elements. Even if the design doesn't require visible labels, it's important to include them for accessibility. Screen readers use them to inform visually impaired users about the purpose of the form fields. You can hide them visually using CSS if necessary.